.vib/node/goss/vars.yaml

```diff
@@ -0,0 +1,29 @@
+binaries:
+  - corepack
+  - node
+  - npm
+  - npx
+  - node-gyp
+  - python
+  - yarn
+files:
+  - paths:
+      - /opt/bitnami/node/lib/node_modules/corepack/package.json
+      - /opt/bitnami/node/lib/node_modules/node-gyp/package.json
+      - /opt/bitnami/node/lib/node_modules/npm/package.json
+      - /opt/bitnami/node/lib/node_modules/yarn/package.json
```

Comment:
I don't think this is necessary. You are already checking the `binaries` and that they appear in `npm list -g`